packages/grpcts/src/LatencyTimer.ts

```diff
@@ -0,0 +1,23 @@
+export interface ILatencyTimer {
+  start: () => ILatency;
+}
+
+export interface ILatency {
+  getValue: () => number;
+}
+
+export class LatencyTimer {
+  public start(): Latency {
+    const now = new Date();
+    return new Latency(now);
+  }
+}
+
+class Latency {
+  constructor(readonly startDate: Date) {}
+
+  public getValue(): number {
```

Comment:
don't like a getter method?
`get value() {} ...`

Reply:
want to leave it as a method, so it's clear form outside that the value is actually calculated when you call it